File: Application/Services/Operations/Outsourced/Dtos/CollectDeliverUpdateDto.cs
using System;
using Application.Services.Operations.Main.Partners.Dtos;
using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.Outsourced.Dtos
{
    public class CollectDeliverUpdateDto : RootBaseDto
    {
        public int? TransporterId { get; set; }
        public PartnerDto Transporter { get; set; }
        // public string SubjectReason { get; set; }
        public string ContactName { get; set; }
        public DateTime Start { get; set; }
        public decimal Price { get; set; }
        public DateTime WasPaid { get; set; }
        public bool Collect { get; set; }
        public bool Deliver { get; set; }
        public bool Other { get; set; }
        public string KindTransport { get; set; }
        public BillingFromDto BillingFrom { get; set; }
        public string TaskOverView { get; set; }
        public DestinyDto Destiny { get; set; }
    }
}

File: Views/src/components/out-sourced/collect-deliver/dto/collect-deliver-update-dto.ts
import { MyUser } from "../../../../components/authentication/dto/my-user";
import { PartnerDto } from "../../../../components/main/partner/dtos/partner-dto";
import { CompanyDto } from "../../../../shared/entities-dtos/company-dto";
import { BillingFromDto } from "./billing-from-dto";
import { DestinyDto } from "./destiny-dto";


export class CollectDeliverUpdateDto {
  id: number;
  companyId: number;
  company: CompanyDto;
  userId: number;
  user: MyUser;
  transporterId: number;
  transporter: PartnerDto;
  // subjectReason: string;
  contactName: string;
  start: Date;
  price: number;
  collect: boolean;
  deliver: boolean;
  other: boolean;
  kindTransport: string;
  billingFromId: number;
  billingFrom: BillingFromDto;
  taskOverView: string;
  destinyId: number;
  destiny: DestinyDto;
}
